st2client/st2client/base.py

```diff
@@ -445,6 +445,6 @@ def _print_client_settings(self, args):
         print("")
         print("Proxy settings:")
         print("---------------")
-        print("HTTP_PROXY: %s" % (os.environ.get("HTTP_PROXY", "")))
-        print("HTTPS_PROXY: %s" % (os.environ.get("HTTPS_PROXY", "")))
+        print("HTTP_PROXY: %s" % (os.environ.get("HTTP_PROXY", os.environ.get("http_proxy"))))
```

Comment:
This means that if neither are specified that None is outputted instead of "", could you use , "" on the inner os.environ.get.

Also, what happens if both are set. I think http_proxy in lowercase generally takes prioiryt, so we should probably change the order so that we check for the lowercase value, and only look for uppercase if its not set.